src/AccessibilityInsights.SharedUx/Properties/Resources.resx

```diff
@@ -156,11 +156,8 @@
   <data name="GeneralActionView_ExecuteAction_Failed" xml:space="preserve">
     <value>Failed!!</value>
   </data>
-  <data name="ButtonContent" xml:space="preserve">
-    <value>I want this Action!!!</value>
-  </data>
-  <data name="TextBlockText" xml:space="preserve">
-    <value>The below Action is not supported yet. If you want this action to be provided in this tool. please click below button.</value>
+  <data name="UnsupportedActionPart1BeforeActionName" xml:space="preserve">
```

Comment:
Can this use one format string rather than 3 separate data values, for better localizability in case the hyperlink and action name need to be in a different order? eg,

```xml
<data name="UnsupportedActionMessage" xml:space="preserve">
  <value>Accessibility Insights for Windows does not yet support the {0} action. To request support for this action, please upvote an existing issue at {1}. If no issue already exists, you can open a new issue (feature request) to request support for this action.</value>
  <comment>{0} is the programmatic ID of the action in question. {1} is a clickable hyperlink whose text is a URL where the user can file issues.</comment>
</data>
```

This would require the code-behind to do something a bit more complicated than the usual `string.Format` to fill in the format specifier, maybe something along the lines of this (rough, untested) code:

```cs
Regex formatStringPlaceholderRegex = new Regex(@"(\{\d})");
string[] formatStringParts = formatStringPlaceholderRegex.Split(formatStringFromResources, 2);
var messageRuns = formatStringParts.Select(part => {
  switch(part) {
    case "{0}": return new Run { Text = actionName; FontWeight = FontWeight.Bold; };
    case "{1}": return new Hyperlink { /* ... etc ... */ };
    default: return new Run { Text = part };
  }
});
this.unsupportedActionMessageTextBlock.Inlines.Clear();
this.unsupportedActionMessageTextBlock.Inlines.AddRange(messageRuns);
```

Reply:
Thanks, @dbjorge! I was following a pattern that we use in a lot of places, but I prefer your pattern. Your "rough, untested" code was remarkably close to the final iteration. Hopefully we'll be able to use similar patterns in other places where we build up text runs via concatenation.